test_minimal_runtime_export_all_modularize fails on windows

See: https://logs.chromium.org/logs/emscripten-releases/buildbucket/cr-buildbucket/8790978322466379425/+/u/Emscripten_testsuite__other_/stdout

```
AssertionError: JS subprocess failed (C:/b/s/w/ir/cache/builder/emscripten-releases/node-v14.18.2-win-x64/node.exe --stack-trace-limit=50 --trace-uncaught --unhandled-rejections=throw C:\b\s\w\ir\x\t\emtest_6n138uii\emscripten_test_other_z3ln08yu\main.mjs): 1.  Output:
fs.js:497
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/test.wasm'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at Module (file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/test.mjs:32:23)
    at main (file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/main.mjs:9:27)
    at file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/main.mjs:13:7
    at ModuleJob.run (internal/modules/esm/module_job.js:183:25)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: '/C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/test.wasm'
}
Thrown at:
    at handleErrorFromBinding (internal/fs/utils.js:331:5)
    at openSync (fs.js:497:3)
    at readFileSync (fs.js:393:35)
    at Module (file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/test.mjs:32:23)
    at main (file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/main.mjs:9:27)
    at file:///C:/b/s/w/ir/x/t/emtest_6n138uii/emscripten_test_other_z3ln08yu/main.mjs:13:7
    at run (internal/modules/esm/module_job.js:183:25)
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -662,6 +662,7 @@ jobs:
             other.test_dot_a_all_contents_invalid
             other.test_emcc_print_search_dirs
             other.test_emcc_print_file_name
+            other.test_minimal_runtime_export_all_modularize
             other.test_pkg_config*"
       - upload-test-results
       # Run a single websockify-based test to ensure it works on windows.

--- a/test/test_other.py
+++ b/test/test_other.py
@@ -1307,6 +1307,7 @@ int f() {
     self.emcc('lib.c', ['-Oz', '-sEXPORT_ALL', '-sLINKABLE', '--pre-js', 'main.js'], output_filename='a.out.js')
     self.assertContained('libf1\nlibf2\n', self.run_js('a.out.js'))
 
+  @no_windows('https://github.com/emscripten-core/emscripten/issues/18596')
   def test_minimal_runtime_export_all_modularize(self):
     """This test ensures that MODULARIZE and EXPORT_ALL work simultaneously.
 

